arrabiata/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+use strum::EnumCount as _;
+
+pub mod column_env;
+pub mod columns;
+pub mod constraints;
+pub mod interpreter;
+pub mod logup;
+pub mod poseidon_3_60_0_5_5_fp;
+pub mod poseidon_3_60_0_5_5_fq;
+pub mod proof;
+pub mod prover;
+pub mod verifier;
+pub mod witness;
+
+/// The maximum degree of the polynomial that can be represented by the
+/// polynomial-time function the library supports.
```

Comment:
I think we need a better name than polynomial time function
There is no context reagarding assymptotic complexity here
I believe the usual name for IVC is application